core/evaluation.py

```
@@ -0,0 +1,396 @@
+    description: str
+    prompt: str
+    difficulty: Difficulty = Difficulty.BASIC
+    verify_type: VerifyType = VerifyType.CONTAINS
+    expected: Any = None
+    expected_tools: List[str] = field(default_factory=list)
+    metadata: Dict[str, Any] = field(default_factory=dict)
+    verification_type: VerifyType = VerifyType.CONTAINS
+    expected_result: Any = None
+
+    def __post_init__(self):
+        """向后兼容：如果没有提供 verify_type，使用 verification_type"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)